src/ngrest/Config.php

```diff
@@ -199,7 +199,23 @@ public function setGroupByField($groupByField)
     {
         $this->_groupByField = $groupByField;
     }
-    
+
+    private $_groupByExpanded;
+
+    public function getGroupByExpanded()
```

Comment:
Please add php docs for getter and setter inlcuding the version number its introduced, @since 1.2.2.1 in this case